root-sync/base/crossplane/rad-lab/gen-ai/xrd-gen-ai.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xradlabgenais.data-science-portal.phac-aspc.gc.ca
+spec:
+  defaultCompositionRef:
+    name: rad-lab-gen-ai
+  group: data-science-portal.phac-aspc.gc.ca
+  names:
+    kind: xRadLabGenAI
+    plural: xradlabgenais
+  claimNames:
+    kind: RadLabGenAIClaim
+    plural: radlabgenaiclaims
+  versions:
+    - name: v1alpha1
+      served: true
+      referenceable: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                # Base Properties
+                rootFolderId:
+                  type: string
+                projectName:
+                  type: string
+                projectId:
+                  type: string
+                projectEditors:
+                  type: array
+                  items:
+                    type: string
+                projectViewers:
+                  type: array
+                  items:
+                    type: string
+                
+                #Gen AI Properties
+                shieldedVm:
+                  type: boolean
+                owners: # "["<name>@gcp.hc-sc.gc.ca",...]"
+                  type: string
+                trustedUsers: # "["<name>@gcp.hc-sc.gc.ca",...]"
+                  type: string
```

Comment:
Do we want to make these a string array and join it in the Composition, or is that too messy?

Reply:
Nested values will definitely be easier as string values since `Workspace` uses key-value pairs as strings. https://marketplace.upbound.io/providers/upbound/provider-terraform/v0.16.0/resources/tf.upbound.io/Workspace/v1beta1#doc:spec-forProvider-vars